index.js

```diff
@@ -132,6 +132,21 @@ function createQuestions(config) {
           : 'Provide a longer description:',
       when: !config.skipQuestions.includes('body')
     },
+    {
+      type: 'confirm',
+      name: 'isBreaking',
+      message: 'Are there any breaking changes?',
+      default: false
+    },
```

Comment:
Can we remove this extra step to fluidify the workflow a little bit? The next step could ask to enter a breaking change if there's any, or leave empty otherwise.

Reply:
Removing this section will affect `function format` when adding `breaking` section.

https://github.com/Clumsy-Coder/cz-emoji/commit/5d69566df5060d505fed0e30cfd1ec8dda389c90#diff-168726dbe96b3ce427e7fedce31bb0bcR175-R177

Is there a way for this to make it work?

Reply:
Wouldn't this work?

```js
const breaking = answers.breakingBody
    ? wrap(`BREAKING CHANGE: ${answers.breakingBody.trim()}`, columns)
    : ''
```

Reply:
This works. Thanks